emails/report_email.py

```diff
@@ -56,38 +55,6 @@ def build(self, report: Report, report_campaign: Campaign):
         return self
 
 
-class ChangeStatusReportEmailTemplate(EmailTemplate):
-    """
-    Report's status is modified
-    Subject: [ValDB][1_2_3_abc] Campaign notification
-    Recipients: forum, report's authors
-    Template: change_status_report_template.html
-
-    Use the same subject to group all emails under the same thread
-    """
-
-    def build(
-        self, report: Report, previous_status: ReportStatus, report_campaign: Campaign
-    ):
-        """
-        build email
-        """
-        self.original_email_id = report_campaign.reference_email_id
-        self.subject = f"[ValDB][{report.campaign_name}] Campaign notification"
-        content_markdown_no_links = parse_attachment_links(content=report.content)
```

Comment:
Following the discussion https://its.cern.ch/jira/browse/PDMVDEV-110, if I understood the logic here, maybe just checking the size of this `content_markdown_no_links` would be enough.